bin/configurer/utils/generate-local-user-mappings.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -eu
+
+swanseauserids=$(docker run -e PGPASSWORD='openidm' --rm --network ccd-network postgres:11-alpine psql --host shared-db  --username openidm --tuples-only  --command "SELECT string_agg(fullobject->>'_id',',') FROM managedObjects WHERE fullobject->>'userName' LIKE '%swansea%';" openidm)
+hillingdonuserids=$(docker run -e PGPASSWORD='openidm' --rm --network ccd-network postgres:11-alpine psql --host shared-db  --username openidm --tuples-only  --command "SELECT string_agg(fullobject->>'_id',',') FROM managedObjects WHERE fullobject->>'userName' LIKE '%hillingdon%';" openidm)
+
+swanseauseridsformatted=$(echo $swanseauserids | tr -d ' ')
+hillingdonuseridsformatted=$(echo $hillingdonuserids | tr -d ' ')
+
+cd ../../../service/src/main/resources
```

Comment:
This is a bit fragile. It won't work when script is executed from location other than scripts dir. As a counter measure all other scripts have:

```
dir=$(dirname ${0})
```

and then use script location to calculate other paths:

```
${dir}/...
```

Reply:
done